src/swap/SwapScreen.test.tsx

```diff
@@ -1634,56 +1737,6 @@ describe('SwapScreen', () => {
     expect(queryByTestId('QuoteResultNeedDecreaseSwapAmountForGasBottomSheet')).toBeFalsy()
   })
 
-  it("should display 'Fund your wallet' bottom sheet with 'Add funds' button when user has zero balance", async () => {
```

Comment:
Is there an opportunity to have a CTA in the new notification for this case?

Reply:
There's actually no analog to this warning in the new inline notification setup. This would previously only show if the user had _no_ balance at all in their wallet. I spoke with Laura about this and she's fine with not having an explicit warning/CTA for this case.